src/index.css

```diff
@@ -0,0 +1,71 @@
+:root {
```

Comment:
- [ ] In the desktop view, the details page does not look like the design given:


### Your design:

![image](https://github.com/CaptainBawa/DocBook-Front-End/assets/57421823/43eec46e-9cb1-4078-8f79-b3c84cb7a359)


### The requested design:

![image](https://github.com/CaptainBawa/DocBook-Front-End/assets/57421823/c5501f9a-78a7-42d2-9e0d-b3e9a27f996b)

Remember:  You need to be able to implement business requirements into a software solution.  This is because, in a job, all software solutions adhering to the given designs will be required

Therefore, kindly make it so your implementation looks a little more like the given design. 💪